clerkinc/javascript
# fix(localizations): German(de-DE), add new translations, fix existing…

## Type of change


## Packages affected

- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [x] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

=> Fails on main branch as well:
```
FAIL shared src/utils/date.test.ts

@clerk/shared:test: Test Suites: 1 failed, 10 passed, 11 total
@clerk/shared:test: Tests:       5 failed, 157 passed, 162 total
@clerk/shared:test: Snapshots:   0 total
@clerk/shared:test: Time:        4.455 s, estimated 5 s
@clerk/shared:test: Ran all test suites.
@clerk/shared:test: npm ERR! Lifecycle script `test` failed with error:
@clerk/shared:test: npm ERR! Error: command failed
@clerk/shared:test: npm ERR!   in workspace: @clerk/shared@0.20.0
@clerk/shared:test: npm ERR!   at location: /Users/manuel/dev/clerk-javascript/packages/shared
@clerk/shared:test: ERROR: command finished with error: command (/Users/manuel/dev/clerk-javascript/packages/shared) npm run test exited (1)
```



- Add german translations for unstable__errors
- Change all 'Kennwort' to 'Passwort' for consistency.
